src/ResourceManager/FrontDoor/Commands.FrontDoor/help/Remove-AzureRmFrontDoorContent.md

```diff
@@ -0,0 +1,136 @@
+---
+external help file: Microsoft.Azure.Commands.FrontDoor.dll-Help.xml
+Module Name: AzureRM.FrontDoor
+online version:
```

Comment:
Follow instructions here to fix CI: https://azuresdkci.westus2.cloudapp.azure.com/job/powershell/8440/artifact/src/Package/ValidateHelpIssues.csv/*view*/.